fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -47,7 +47,7 @@ def build_upload_command(username, password, source = "/tmp", provider_short_nam
       not_implemented(__method__)
     end
 
-    def build_verify_command(username, password, source = "/tmp", provider_short_name = "", jwt = nil)
+    def build_verify_command(username, password, source = "/tmp", provider_short_name = "", jwt = nil, platform = nil, api_key = nil)
```

Comment:
[nits] I think it's better to use `**kwargs` in the method signatures of the abstract method.

`jwt` is only needed for ITMSTransporter, and `platform` and `api_key` are only needed for altool.
So it's not better to have all parameters on the super method for code readability.

But this point is not important.

Reply:
Just modified the abstract method to use `**kwargs` 👍